Tribler/Main/vwxGUI/list_body.py

```diff
@@ -763,6 +763,9 @@ def OnCollapse(self, item=None, raise_events=True, from_expand=False):
 
     @warnWxThread
     def OnChange(self, scrollToTop=False):
+        if GUIUtility.getInstance().utility.abcquitting:
```

Comment:
This is the tricky part. `if self` doesn't work for this problem. The issue is that when this `OnChange()` is called by an wx event and `ListBody` is not dead yet. So `_SetupAfter()` has scheduled but somehow not executed before `ListBody` is dead. After we exit the MainLoop and wake up the MainThread again (for the shutdown splash window), this `_SetupAfter()` got scheduled and `PyDeadObject` appears.

So the only way I found is to use this `GUIUtility.getInstance().utility.abcquitting` to avoid calling `SetupScrolling()`.